### PR TITLE
Bumping gradle to 2.7

### PR DESCRIPTION
**Pull request by marcingrzejszczak**

### CODE REVIEW

**Comment by marcingrzejszczak** [comment]
CC @szpak

**Comment by marcingrzejszczak** [comment]
@szpak - you have to bump up `micro-common-release` libs

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-bin.zip
```
Comment (hanskoff) [inline gradle/wrapper/gradle-wrapper.properties]: awesome,

File: build.gradle [hunk 2]
```diff
@@ -11,7 +11,8 @@ buildscript {
         classpath "org.springframework.boot:spring-boot-gradle-plugin:$springBootVersion"
         classpath 'com.ofg:uptodate-gradle-plugin:+'
         classpath "com.ofg:micro-common-release:0.1.18"
-        classpath "io.spring.gradle:dependency-management-plugin:0.2.1.RELEASE"
+        classpath "io.spring.gradle:dependency-management-plugin:0.5.3.RELEASE"
+        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:+"
```
Comment (hanskoff) [inline build.gradle]: :+1:

**Pull request merged**